examples.json

```diff
@@ -123,7 +123,13 @@
         "slug": "validate-options",
         "jsbinId": "qafefu",
         "keywords": "api-check apiCheck"
-      }
+      },
+      {
++        "title": "ng-currency with maxlength",
```

Comment:
Thanks for the PR! Example looks good. These `+` signs make the JSON invalid. Please remove them. Also, could you please move this to the bottom of the `Integrations` section? Thanks!